### DISCUSSION TITLE
Is it possible to use this flake with devenv?

### DISCUSSION THREAD

**Discussion by dudeofawesome** [juspay/services-flake#347] [Q&A]
I'd like to augment the services available in devenv with the many services available from this project.

**Comment by shivaraj-bh** [comment]
Try following this https://devenv.sh/guides/using-with-flake-parts/. 

Once you have `devenv` with `flake-parts` in a flake, add `services-flake` to that same flake and you can simply refer to the package exported by it in your `devenv` configuration via `self’.packages.<process-compose-project-name>`.

I haven’t tried this. Let me know how it goes

**Reply by srid** [reply]
Tangential: https://github.com/cachix/devenv/issues/75
